external/dynamo/generalized/processor.py

```
@@ -0,0 +1,429 @@
+                decision_id=decision_id,
+                latency_ms=float(latency_ms),
+                success=bool(success),
+                tokens_in=int(tokens_in),
+                tokens_out=int(tokens_out),
+                finish_reason=finish_reason or "",
+            )
+            stream = await self.router_feedback_client.generate(fb.model_dump())
+            async for _ in stream:
+                pass
+        except Exception as e:
+            logger.exception("Failed to send router feedback: %s", e)
```

Ruff (0.14.8)

Redundant exception object included in `logging.exception` call

(TRY401)